src/middlewared/middlewared/alembic/versions/12.0/2020-03-30_13-05_ntpd_user.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Manipulating built-in users/groups through DB migrations won't work good for cross-platform app.

We use assets/account/builtin/<platform>/(group|passwd) file instead. `account.sync_builtin` method does all the job.